hs-bindgen-runtime/src/HsBindgen/Runtime/CEnum/General.hs

```diff
@@ -0,0 +1,305 @@
+{-# LANGUAGE TypeFamilies #-}
+
+module HsBindgen.Runtime.CEnum.General (
```

Comment:
Could consider changing naming convention for qualified imports, would make this easier perhaps. No strong preference.